src/structure/model.jl

```diff
@@ -0,0 +1,1089 @@
+serial(disordered_atom::DisorderedAtom) = serial(defaultatom(disordered_atom))
+atomname(disordered_atom::DisorderedAtom) = atomname(defaultatom(disordered_atom))
+altlocid(disordered_atom::DisorderedAtom) = defaultaltlocid(disordered_atom)
+resname(disordered_atom::DisorderedAtom) = resname(defaultatom(disordered_atom))
+chainid(disordered_atom::DisorderedAtom) = chainid(defaultatom(disordered_atom))
+resnumber(disordered_atom::DisorderedAtom) = resnumber(defaultatom(disordered_atom))
+inscode(disordered_atom::DisorderedAtom) = inscode(defaultatom(disordered_atom))
+x(disordered_atom::DisorderedAtom) = x(defaultatom(disordered_atom))
+y(disordered_atom::DisorderedAtom) = y(defaultatom(disordered_atom))
+z(disordered_atom::DisorderedAtom) = z(defaultatom(disordered_atom))
+coords(disordered_atom::DisorderedAtom) = coords(defaultatom(disordered_atom))
+occupancy(disordered_atom::DisorderedAtom) = occupancy(defaultatom(disordered_atom))
+tempfac(disordered_atom::DisorderedAtom) = tempfac(defaultatom(disordered_atom))
+element(disordered_atom::DisorderedAtom) = element(defaultatom(disordered_atom))
+charge(disordered_atom::DisorderedAtom) = charge(defaultatom(disordered_atom))
+
+isdisorderedatom(::DisorderedAtom) = true
+atomid(disordered_atom::DisorderedAtom) = atomid(defaultatom(disordered_atom))
+
+# Constructor acts as a setter for the default alt loc ID
+function DisorderedAtom(disordered_atom::DisorderedAtom, default::Char)
+    @assert default in altlocids(disordered_atom) "The new default alternative location ID must be present in the atom"
+    return DisorderedAtom(disordered_atom.alt_loc_ids, default)
+end
+
+# These coordinate setters only set the default atom coordinates
+x!(disordered_atom::DisorderedAtom, x::Real) = x!(defaultatom(disordered_atom), x)
+y!(disordered_atom::DisorderedAtom, y::Real) = y!(defaultatom(disordered_atom), y)
+z!(disordered_atom::DisorderedAtom, z::Real) = z!(defaultatom(disordered_atom), z)
+coords!(disordered_atom::DisorderedAtom, coords::Vector{Float64}) = coords!(defaultatom(disordered_atom), coords)
+
+"""
+Get a descriptive residue ID for an `AbstractAtom` or `AbstractResidue`. Format
+is residue number then insertion code with \"H_\" in front for hetero residues.
+If `full` equals true the chain ID is also added after a colon. Examples are
+\"50A\", \"H_20\" and \"10:A\".
+"""
+function resid(element::Union{AbstractResidue, AbstractAtom}; full::Bool=false)
+    if ishetero(element)
+        if full
+            inscode(element) == ' ' ? "H_$(resnumber(element)):$(chainid(element))" : "H_$(resnumber(element))$(inscode(element)):$(chainid(element))"
+        else
+            inscode(element) == ' ' ? "H_$(resnumber(element))" : "H_$(resnumber(element))$(inscode(element))"
+        end
+    else
+        if full
+            inscode(element) == ' ' ? "$(resnumber(element)):$(chainid(element))" : "$(resnumber(element))$(inscode(element)):$(chainid(element))"
```

Comment:
These lines are too long. Using `if inscode(element) == ' ' ... else ... end` instead of the ternary operator will break long lines here.